### ISSUE TITLE
Why the reconstructed image in iradon transformation is multiplied by 1/2

### ISSUE THREAD

**Issue by alwaysuu** [scikit-image/scikit-image#6916]
### Description:

Hello, I recently encountered the same problem as #6174 , which is that **Why the Ramp filter is multiplied by 2 in iradon transform**. After carefully checking the discussion in #6174 and the code in skimage/transform/radon_transform.py, I think there is a bug in iradon function.

First of all, I agree with @z0gSh1u that multiplying Ramp filter by 2 is not necessary. But if we drop this multiplier, the reconstructed image will have just the half intensity of ground truth, **which is actually caused by the code in iradon function and I think it is wrong**.

Please see line 305 in skimage/transform/radon_transform.py

```python
return reconstructed * np.pi / (2 * angles_count)
``` 

Why the reconstructed image in iradon transformation is multiplied by $\frac{1}{2}$? I think having this $\frac{1}{2}$  is wrong. Below is my explanation:

We know that the final stage of FBP is to sum all back projections from different views, so as to obtain the reconstructions. In theory, it is 

![image](https://user-images.githubusercontent.com/48625631/236612364-3b52dcac-25d0-4e07-8ef5-c468873830fd.png)
(from digital image processing, 4th edition, page 387)

In practice, we have to deal with these two integrations with numerical summations. The inner integration is done by convolution in the spatial domain (or multiplication in the frequency domain).  For the outer integration, we know that

$$
\int_0^{\pi} f(\theta) d\theta = \int_0^{\pi}  f(\theta) \frac{\pi}{\pi} d\theta = \pi \int_0^{\pi}  f(\theta) p(\theta) d\theta = \pi E_{p(\theta)}[ f(\theta)] 
$$

where $f(\theta)$ denotes the inner integration,  $\theta \sim p(\theta)=\frac{1}{\pi}$ follows uniform distrubution in the range $[0, \pi]$.

We can therefore use Mento Carlo integration to approximate the above expectation, which results is

$$
\pi E_{p(\theta)}[f] \approx \pi \frac{1}{N} \sum_{\theta = 0}^\pi f(\theta)
$$

As we can see from the above equation, no multiplier $\frac{1}{2}$ is needed and the code in iradon is not correct. 

In the current version of the code, $2$ in Ramp filter and $\frac{1}{2}$ in iradon function cancel out, which accidentally induces the right answer. But I think the pipeline is not correct and will bring misunderstanding.

**Comment by alwaysuu** [comment]
Sorry for the late reply. 

Yes, I think we should drop these two multipliers. As iradon function in fact implements the filtered back-projection algorithm, which consists of two steps, i.e., filtering and back-projection, respectively. The two steps are separate and independent. Although the final result is correct, these two multipliers may confuse users and lead misunderstanding.

**Comment by stefanv** [comment]
Due diligence would be to:

- Carefully examine:

```
    .. [1] AC Kak, M Slaney, "Principles of Computerized Tomographic
           Imaging", IEEE Press 1988.
    .. [2] B.R. Ramesh, N. Srinivasa, K. Rajgopal, "An Algorithm for Computing
           the Discrete Radon Transform With Some Applications", Proceedings of
           the Fourth IEEE Region 10 International Conference, TENCON '89, 1989
```

- Compare with the output of another piece of software, as a sanity check.

I briefly scanned the papers, and I don't see the normalization factors there.

**Comment by josteinbf** [comment]
> /cc @josteinbf

Thanks -- I haven't looked at a radon transform in nearly a decade, so I'm afraid I won't be of much help here. Hope I didn't create any bugs here :)

**Comment by stefanv** [comment]
@josteinbf Same for me, I think. But I thought you may have some intuition around the scaling. Transforms often have scaling factors like these that are typically split symmetrically.

**Comment by mkcor** [comment]
Hi all,

I'll try to dive into this again. What I remember is that I once (back at #6174) made sense of these factors by considering that the integrals (well, sums) were taken over either the half interval [0, N] or the full interval [-N, N]. Along these lines, indeed, @stefanv:

> Transforms often have scaling factors like these that are typically split symmetrically.

From a quick search, I found this [PyTorch implementation](https://github.com/drgHannah/Radon-Transformation) which uses the same scaling factors as current scikit-image:

* 2 for the [ramp filter](https://github.com/drgHannah/Radon-Transformation/blob/fdee7cee6179b40aed4f492c5aa0ed3916082d25/radon_transformation/filter.py#L11):

```py
return torch.tensor(2 * np.real(np.fft.fft(f)) )
```

* 1/2 for the [reconstructed image](
https://github.com/drgHannah/Radon-Transformation/blob/fdee7cee6179b40aed4f492c5aa0ed3916082d25/radon_transformation/radon.py#L113):

```py
return reconstructed  * np.pi / (2 * self.n_angles)
```

**Comment by mkcor** [comment]
Dear @drgHannah,

I'm pinging you as the author of the PyTorch implementation I mentioned right above. Do you happen to have based your implementation on scikit-image's? If so, my considerations would be circular...

**Comment by mkcor** [comment]
I admit that the 1/2 factor wouldn't be expected, since the angle runs from 0 to pi (it would be expected if the angle ran from 0 to 2*pi): 

https://github.com/scikit-image/scikit-image/blob/3998940b700cdfa7f710201f05271797f92430ac/skimage/transform/radon_transform.py#L192-L194

see equations (72) vs. (71) in [chapter 3](https://engineering.purdue.edu/~malcolm/pct/CTI_Ch03.pdf) of reference [1].

Now, I also wonder why

https://github.com/scikit-image/scikit-image/blob/3998940b700cdfa7f710201f05271797f92430ac/skimage/transform/radon_transform.py#L292

wouldn't be
```py
        t = xpr * np.cos(angle) + ypr * np.sin(angle)
```
instead.
